docs/install.rst

```diff
@@ -0,0 +1,31 @@
+.. _topics-install:
+
+=======
+Install
+=======
+
+.. note:: As a general rule, it's recommended to use an isolation tool such as ``virtualenv``.
+
+Prerequisites:
+
+* ducktape uses paramiko, which depends upon cryptography (https://cryptography.io/en/latest/installation/), which has non-python external requirements
+
+    * OSX should just work
+
+    * Ubuntu::
+
+        $ sudo apt-get install build-essential libssl-dev libffi-dev python-dev
+
+    * Fedora and RHEL-derivatives::
+
+        $ sudo yum install gcc libffi-devel python-devel openssl-devel
+
+* Install ducktape::
+
+    pip install ducktape
+
+
+
+* On macOS you may need to ``brew install openssl`` and then install using::
```

Comment:
Hmm, I actually added this note to the README and I'm surprised I didn't just put it up with the OSX section. Maybe consolidate. Unfortunately, I think it depends on the version of macOS, though maybe at this point we should just assume people are on the latest version?